.github/workflows/integrationtests.yaml

```diff
@@ -114,5 +114,5 @@ jobs:
 
       - name: Download opentelemetry javaagent.
         run: wget -q https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases/download/v1.12.0/opentelemetry-javaagent.jar
-      - name: Micrometer Integration Tests.
-        run: ./gradlew -p core/micrometer test -DincludeTags="OpenTelemetryIntegrationTest"
+      - name: Opentelemetry Integration Tests.
+        run: ./gradlew test -DincludeTags="OpenTelemetryIntegrationTest"
```

Comment:
I did that because when using `./gradlew -p system-tests/test test -DincludeTags="OpenTelemetryIntegrationTest"` I had an EdcInjectionException. For example not able to find implementation of `ContractDefinitionStore` when starting the module system-tests:runtimes:file-transfer-provider. Another solution would be to import the file-transfer modules with `testRuntimeOnly` instead of `testCompileOnly` but I assumed the first solution was better. Let me know if you have a better idea.

Reply:
Just FYI... I see this was changed in this upstream PR https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/pull/911

Reply:
As discussed with @algattik we created a bug: https://github.com/agera-edc/DataSpaceConnector/issues/181 and a PR to fix the problem. I merged the fix to this PR for now.